packages/config/config/devices/0x024e/templates/zconnect_template.json

```diff
@@ -0,0 +1,131 @@
+{
+	"base_0-65000": {
+		"valueSize": 2,
+		"minValue": 0,
+		"maxValue": 65000,
+		"unsigned": true
+	},
+	"base_1-65000": {
+		"$import": "#base_0-65000",
+		"minValue": 1
+	},
+	"base_-60-1000": {
+		"valueSize": 4,
+		"minValue": -60,
+		"maxValue": 1000,
+		"unsigned": false
+	},
+	"interval_1-65000": {
+		"$import": "#base_1-65000",
+		"unit": "0.1 seconds"
+	},
+	"interval_1-65000_disable": {
+		"$import": "#base_0-65000",
+		"unit": "0.1 seconds",
+		"defaultValue": 0,
+		"options": [
+			{
+				"label": "Disable",
+				"value": 0
+			}
+		]
+	},
+	"interval_1-600_disable": {
+		"$import": "#interval_1-65000_disable",
+		"unit": "seconds",
+		"maxValue": 600
+	},
+	"relay_mode_selection": {
+		"label": "Relay mode selection",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 2,
+		"defaultValue": 0,
+		"options": [
+			{
+				"label": "Normal mode - Closed contacts when On, open contacts when Off.",
+				"value": 0
+			},
+			{
+				"label": "Reverse mode - Closed contacts when Off, open contacts when On.",
+				"value": 1
+			},
+			{
+				"label": "Strobe mode - Strobe when On, open contacts when Off.",
+				"value": 2
+			}
+		]
+	},
+	"sensor_relay_control": {
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 4,
+		"defaultValue": 0,
+		"options": [
+			{
+				"label": "Sensors don't control relay.",
+				"value": 0
+			},
+			{
+				"label": "Sensor 1 controlls relay.",
+				"value": 1
+			},
+			{
+				"label": "Sensor 2 controlls relay.",
+				"value": 2
+			},
+			{
+				"label": "Sensor 3 controlls relay via S-Bus.",
+				"value": 3
+			},
+			{
+				"label": "Sensor 4(temperature) controlls relay via S-Bus.",
+				"value": 4
+			}
+		]
+	},
+	"sensor_processing_options": {
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 8,
+		"defaultValue": 0,
+		"options": [
+			{
+				"label": "Do not react on On/Off actions.",
+				"value": 0
+			},
+			{
+				"label": "Send command On during time without action and send command Off on event/trigger.",
```

Comment:
Try to shorten these if you can. They display in drop-downs so they get unruly when long.

Reply:
Sounds good. Most of these I cut and pasted from the manual.

Reply:
Yeah, I know. It's nothing you did wrong. Manufacturers are terrible. We're trying to progressively improve our files.